src/RoleDB.php

```diff
@@ -163,25 +109,4 @@ public function getRoleKeyIds(string $roleName)
         $roleInfo = $this->getRoleInfo($roleName);
         return $roleInfo['keyids'];
     }
-
-    /**
-     * Gets the threshold required for a given role.
-     *
-     * @param string $roleName
-     *    The role name.
-     *
-     * @return integer
-     *     The threshold number of signatures required for the role.
-     *
-     * @throws \Exception
-     *     Thrown if the role does not exist.
-     */
-    public function getRoleThreshold(string $roleName)
-    {
-        if (! $this->roleExists($roleName)) {
-            throw new \Exception("Role does not exist: $roleName");
-        }
-
-        return $this->roles[$roleName]['threshold'];
-    }
```

Comment:
More mostly dead code (to discuss separately as above).

Reply:
Restored for now.